web/src/main/java/edu/gemini/lch/web/app/windows/alarm/StatusPanel.java

```diff
@@ -237,7 +237,7 @@ public BeamCollision() {
                     // get first collision (sorted by time) and check if we need to react
                     // we need to react if a) laser is on sky and b) other telescope has priority
                     LtcsService.Collision c = collisions.get(0);
-                    Boolean needToReact = snapshot.getEpicsSnapshot().isOnSky() && !c.geminiHasPriority();
+                    boolean needToReact = snapshot.getEpicsSnapshot().isOnSky() && !c.geminiHasPriority();
```

Comment:
Is this related to the fix?

Reply:
No... it was just irritating me like nails on a chalkboard. I can reverse it if you want.

Reply:
don't worry I was just curious